protocol-definitions/VectorCollection.yaml

```
@@ -242,3 +242,27 @@ methods:
           doc: |
             Zero or more VectorSearchResult values.
 
+  - id: 9
+    name: optimize
+    since: 2.8
+    doc: |
+      Optimize index.
+    request:
+      retryable: true
+      partitionIdentifier: none
+      params:
+        - name: name
+          type: String
+          nullable: false
+          since: 2.8
+          doc: |
+            Name of the Vector Collection.
+        - name: indexName
+          type: String
+          nullable: true
+          since: 2.8
+          doc: |
+            Name of the Index to optimize.
+            A null value signifies the optimization of the first and only index within the collection.
```

```suggestion
            A null value triggers the optimization of the only index within the collection.
```
if it is only, it is first